hddm/cnn/train_detector.py

```diff
@@ -0,0 +1,390 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import sys
+import tempfile
+import os, glob, pickle
+import tensorflow as tf
+from .config import *
+from .tf_data_handler import inputs
+import numpy as np
+import tqdm, time
+import matplotlib.pyplot as plt
+import math
+from textwrap import wrap
+
+class cnn_model_struct:
```

Comment:
classes should be `CamelCase`: `CNNModelStruct`

Reply:
done